docs/labs/1-JavaEclipse.md

```diff
@@ -26,6 +27,55 @@ You can do this by:
 1. Visiting your add and remove programs setting in windows (or applications on a mac)
 2. Searching for any java programs
 3. **Removing** any existing java installations.
+4. Or you can check out these instruction.
+  a. For **Window OS**:
+  **Windows 11** - Uninstall Programs
+  Click **Start**
+  Select **Settings**
+  Select **System**
+  Select **Apps - Apps & features**
+  Select the program to uninstall and then click its Uninstall button.
+  Respond to the prompts to complete the uninstall
+
+  **Windows 10** - Uninstall Programs
+  Click **Start**
+  Select **Settings**
+  Select **System**
+  Select **Apps & features**
+  Select the program to uninstall and then click its Uninstall button.
+  Respond to the prompts to complete the uninstall
```

Comment:
Let's combine these two.
Also, please remove the instructions for Windows 8, 7, Vista & XP.

Also, please place a `- ` before each step